code/headers/uarm_gcode_generator_c.hpp

```diff
@@ -22,7 +22,7 @@ namespace r2d2::robot_arm {
          * @return char* to buffer
```

Comment:
Why is this file called uarm_gcode_generator```_c```.hpp why not just uarm_gcode_generator